src/modal/modal.ts

```diff
@@ -31,4 +31,9 @@ export class NgbModal {
    * @since 3.1.0
    */
   dismissAll(reason?: any) { this._modalStack.dismissAll(reason); }
+
+  /**
+   * Indicates if there are currently any open modal windows in the application.
+   */
+  hasOpenWindows(): boolean { return this._modalStack.hasOpenWindows(); }
```

Comment:
Some nitpicking here, but I would have named it `hasOpenModals()`
Even your commit message is using open modals and not windows

Reply:
Ok, thnx, will change it to `hasOpenModals()`